dtfs-central-api/src/v1/controllers/utilisation-report-service/tfm-put-utilisation-report-status.controller.ts

```diff
@@ -0,0 +1,135 @@
+import { Request, Response } from 'express';
+import {
+  Collection,
+  DeleteResult,
+  ObjectId,
+  UpdateResult,
+} from 'mongodb';
+
+import {
+  PutReportStatusRequestBody,
+  ReportDetails,
+  ReportFilter,
+  ReportStatus,
+} from './types';
+import * as db from '../../../drivers/db-client';
+import { DB_COLLECTIONS } from '../../../constants/dbCollections';
+
+const REPORT_STATUS: Record<ReportStatus, ReportStatus> = {
+  PENDING_RECONCILIATION: 'PENDING_RECONCILIATION',
+  REPORT_NOT_RECEIVED: 'REPORT_NOT_RECEIVED',
+};
```

Comment:
I originally did this as an enum, but I don't know of a way to tell an enum that it needs to match the type of `ReportStatus`

Reply:
Can you make the type `ReportStatus` an enum and then remove this variable?

Reply:
@toddym42 I can't remember exactly what we discussed but what was the reason we had for not using enums?

Reply:
Details sent over slack